### PR TITLE
Translate to English

### PR DESCRIPTION
**Pull request by sogamoso**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -76,33 +76,33 @@ <h1 class='logo'>
 <a class='social__link' href='http://twitter.com/RubyConfCo' taget='_blank'>Twitter</a>
 </div>
 <div class='intro'>
-<h2 class='intro__title'>&gt;&gt; Hola ¿estás listo?</h2>
+<h2 class='intro__title'>&gt;&gt; Are you ready?</h2>
```
Comment (julian-amaya) [inline docs/index.html]: Should be `Hey, are you ready?` (related to the puts)

File: docs/index.html [hunk 2]
```diff
@@ -76,33 +76,33 @@ <h1 class='logo'>
...
 <p class='intro__description'>
-Queremos que seas parte de <strong>RubyConf Colombia 2017</strong>. Nuestro equipo organizador está trabajando activamente en seleccionar el mejor contenido para hacer un evento inolvidable.
+Be part of <strong>RubyConf Colombia 2017</strong>. The organizing team is hard at work to select the best content and make this the best an experience the best Ruby conference in Colombia ever.
```
Comment (julian-amaya) [inline docs/index.html]: I think it is better like this:
Be part of <strong>RubyConf Colombia 2017</strong>. The organizing team is working hard selecting the best content to organize the best Ruby conference in Colombia ever.

File: docs/index.html [hunk 3]
```diff
@@ -76,33 +76,33 @@ <h1 class='logo'>
...
 <strong>9</strong>
-de 2017
+/ 2017
 </span>
 </p>
 </div>
 <div class='newsletter'>
 <div class='newsletter__description'>
-Si quieres recibir información sobre la conferencia, regístrate y te estaremos contactando con detalles sobre la organización del evento.
+If you want to recieve info about the conference, join our mailing list. No spam, we promise.
```
Comment (julian-amaya) [inline docs/index.html]: typo: *receive*

File: source/index.html.haml [hunk 4]
```diff
@@ -29,32 +29,32 @@ title: RubyConf Colombia 2017
...
-            puts "Hola ¿estás listo?"
+            puts "Hey, are you ready?"
   .column.right-column
     .top
       .social
         %a.social__link{:href => "http://facebook.com/RubyConfCo", :taget => '_blank'} Facebook
         %a.social__link{:href => "http://twitter.com/RubyConfCo", :taget => '_blank'} Twitter
       .intro
-        %h2.intro__title &gt;&gt; Hola ¿estás listo?
+        %h2.intro__title &gt;&gt; Are you ready?
```
Comment (julian-amaya) [inline source/index.html.haml]: Same comments as in the index file

**Review by julian-amaya — CHANGES_REQUESTED** [review]

**Comment by sogamoso** [comment]
@julian-amaya done!

File: docs/index.html [hunk 5]
```diff
@@ -65,44 +65,46 @@ <h1 class='logo'>
...
+If you want to receive info about the conference, join our mailing list.
+<br>
+No spam, we promise.
 </div>
 <form action='//rubyconf.us11.list-manage.com/subscribe/post?u=7b7c2e5ead7427750ced71f09&amp;id=22ba1d80e6' class='form row newsletter__form mailchimp-form' enctype='multipart/form-data' method='post' name='mc-embedded-subscribe-form' target='_blank'>
 <div class='newsletter__field'>
-<input class='newsletter__field_input' id='EMAIL' name='EMAIL' placeholder='tu@email.com' type='email'>
-<button class='newsletter__field_button'>Suscribirse</button>
+<input class='newsletter__field_input' id='EMAIL' name='EMAIL' placeholder='your@email.com' type='email'>
+<button class='newsletter__field_button'>Suscribe</button>
```
Comment (julian-amaya) [inline docs/index.html]: Typo *subscribe*

**Review by julian-amaya — CHANGES_REQUESTED** [review]
Just one more type but I think we're good

File: docs/index.html (hunk 5 above)
Comment (sogamoso) [reply docs/index.html]: Done :)

**Review by julian-amaya — APPROVED** [review]

**Pull request merged**